docs/vectorizer-quick-start.md

```diff
@@ -0,0 +1,130 @@
+# Vectorizer quick start
+
+This page shows you how to create a vectorizer in a self-hosted Postgres instance, then use 
+a pgai vectorizer worker to create embeddings from data in your database. You view the 
+embeddings with a standard SQL query. 
```

Comment:
```suggestion
This page shows you how to create a vectorizer in a self-hosted Postgres instance, then use 
the pgai vectorizer worker to create embeddings from data in your database. To finish off we show how simple it is to do a semantic search query on the embedded data in one query!
```

Viewing the embeddings is kinda lame, we actually do semantic search which is imo the aha effect of this guide.